src/injector.ts

```diff
@@ -156,15 +160,11 @@ function plugin(
 
 				let usedProps: string[] = [];
 
-				if (!includeUnusedProps) {
```

Comment:
Just because I don't want to include unused props by default doesn't meant I might actually want to include them via `shouldInclude`.

Reply:
Fixed in https://github.com/merceyz/typescript-to-proptypes/commit/277258ddc73c3da816aba6fccb739c69dfe8e83a